docs/components/SquarePaymentForm.md

```diff
@@ -23,7 +23,7 @@ Please view the [Payment Form Data Models](https://docs.connect.squareup.com/api
 |---|---|---|
 |apiWrapper|string|Internal variable: used for logs<br/><br/>**Default Value:** `"reactjs/0.4.0"`|
 |applicationId|string|<b>Required for all features</b><br/><br/>Identifies the calling form with a verified application ID generated from the Square Application Dashboard|
-|cardNonceResponseReceived|(errors: [SqError], nonce: string, cardData: SqCardData, buyerVerificationToken: string) => void|<b>Required for all features</b><br/><br/>Invoked when payment form receives the result of a nonce generation request. The result will be a valid credit card or wallet nonce, or an error.|
+|cardNonceResponseReceived|(errors: [SqError], nonce: string, cardData: SqCardData, buyerVerificationToken: string, billingContent: SqContact, shippingContact: SqContact, shippingOption: SqShippingOption) => void|<b>Required for all features</b><br/><br/>Invoked when payment form receives the result of a nonce generation request. The result will be a valid credit card or wallet nonce, or an error.|
```

Comment:
should this line say `billingContact` instead of `billingContent`?

Reply:
Yes!! Nice catch!